spec/expression.dd

```diff
@@ -4,11 +4,31 @@ $(SPEC_S Expressions,
 
 $(HEADERNAV_TOC)
 
+$(P This chapter defines the syntax, order of evaluation, and semantics of expressions. An expression is a sequence
+of operators and operands that specifies an evaluation.)
+
```

Comment:
"An expression is a sequence of operators and operands that specifies an evaluation.
The syntax, order of evaluation, and semantics of expressions are as follows."

(I dislike things that start with "This chapter defines...". It seems redundant.)